javascript-light-bulb/main.js

```diff
@@ -0,0 +1,17 @@
+var clickCount = 0;
+
+var $circle = document.querySelector('.light-on');
+var $body = document.querySelector('body');
+
+function lightOnOff(event) {
+  clickCount++;
```

Comment:
You aren't using the click count here for anything other than checking if the value is even or odd. In this case, a variable holding a boolean value would be a better way of keeping track of if the light is currently on or not. You should probably name the variable something like `isLightOn`